solidago/src/solidago/mehestan/individual/indiv_matrix_inversion.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+import pandas as pd
+
+from .base import IndividualScoresAlgorithm
+
+DEFAULT_ALPHA = 1.0  # Signal-to-noise hyperparameter
+
+
+class MatrixInversionInvididualAlgo(IndividualScoresAlgorithm):
```

Comment:
Can we quickly brainstorm another model name for this one? A name that would describe the model, not the technique. Otherwise CBT should be called `CoordinateDescentIndividualAlgorithm`

Reply:
I agree this name is not great. I'm open to suggestions.

Reply:
Following up yesterday's discussion, I'd propose: `HookeIndividualScoresAlgorithm` (my prefered option, as in https://fr.wikipedia.org/wiki/Loi_de_Hooke) or `QuadraticIndividualScoresAlgorithm`

I also like better **not** to shorten Algorithm -> Algo